src/GraphQL.Tests/GraphQLValuesCacheTests.cs

```diff
@@ -25,6 +31,12 @@ public void GetLong()
         GraphQLValuesCache.GetLong("-1000000").ShouldBe(-1000000L);
         GraphQLValuesCache.GetLong("1000000").ShouldBe(1000000L);
 
-        Should.Throw<FormatException>(() => GraphQLValuesCache.GetLong("a")).Message.ShouldBe("Input string was not in a correct format.");
+#if NET7_0_OR_GREATER
```

Comment:
It is unfortunate that MS has changed the FormatException message from .NET 6 -> .NET 7. Maybe the strict exception message check could be removed and just check for throwing `FormatException`. In that case the nasty `ifdef` could be eliminated.